mlir-tensorrt/executor/test/lib/BufferizationTestPass.cpp

```diff
@@ -53,22 +53,32 @@ class ExecutorBufferizationTestPass
     }
   }
 };
+
+struct PlanBufferizationPipelineCliOpts
```

Comment:
Plan bufferization is not part of the `executor` project. It looks like all the changes here can be removed since `enableNonDPSReturns` is not actually used anywhere here.